### PR TITLE
add typings for chayns version 0.14.0

### PR DESCRIPTION
**Pull request by HenningKuehl**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/chayns/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,938 @@
+// Type definitions for chayns 0.14
+// Project: https://gitlab.tobit.com/chayns-api/chayns-typings
```
Comment (mhegazy) [inline types/chayns/index.d.ts]: the link seems invalid, is this the correct link: https://github.com/TobitSoftware/chayns-js?

**Review by mhegazy — CHANGES_REQUESTED** [review]

File: types/chayns/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,938 @@
...
+// Definitions by: Henning Kuehl <https://github.com/HenningKuehl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/**
+ * Definition file for chayns v3.1
```
Comment (mhegazy) [inline types/chayns/index.d.ts]: so what is `0.14` above?

File: types/chayns/index.d.ts (hunk 1 above)
Comment (HenningKuehl) [inline types/chayns/index.d.ts]: That is our internal link for the typings project. You are right, the typings are for that project.

File: types/chayns/index.d.ts (hunk 2 above)
Comment (HenningKuehl) [inline types/chayns/index.d.ts]: The JavaScript API has version 3.1, but our typings are not completed yet.

File: types/chayns/index.d.ts (hunk 1 above)
Comment (mhegazy) [inline types/chayns/index.d.ts]: not sure what you mean by "internal". these packages are consumed by everyone, and the urls should be available to all.

File: types/chayns/index.d.ts (hunk 2 above)
Comment (mhegazy) [inline types/chayns/index.d.ts]: The version in the header is what ends up as the version of @types package. the policy is that this version matches the major.minor of the package it defines, any updates to the definition file will have a build number pump.

File: types/chayns/index.d.ts (hunk 1 above)
Comment (HenningKuehl) [inline types/chayns/index.d.ts]: Yes, already understand, that was a mistake of mine.

File: types/chayns/index.d.ts (hunk 2 above)
Comment (HenningKuehl) [inline types/chayns/index.d.ts]: So the version of the typings must equal the version of the JavaScript API?

Comment (mhegazy) [inline types/chayns/index.d.ts]: yes. major.minor

**Pull request merged**